src/monda/tests/sorad_retrieval_qc_plot_test.py

```diff
@@ -307,5 +257,7 @@ def parse_args():
     if not os.path.isdir(args.target):
         os.mkdir(args.target)
 
-    response = run_example(args.platform, args.start_time, args.end_time, args.bbox, args.target, args.algorithm.lower(),
-                           args.output_radiance, args.output_metadata, args.output_rrs, args.output_plots)
+    # response = run_example(args.platform, args.start_time, args.end_time, args.bbox, args.target, args.algorithm.lower(),  # use if running in terminal
```

Comment:
tidy up commented lines - either they are needed or not..

Reply:
done